client/src/components/NavBar/AuthBars/LoggedInBar.tsx

```diff
@@ -4,15 +4,77 @@ import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import AuthMenu from '../../AuthMenu/AuthMenu';
+import Notifications from './Notifications/Notifications';
+import { useState, useEffect } from 'react';
+import getUnread from '../../../helpers/APICalls/getAllUnreadNotifications';
+import readNotification from '../../../helpers/APICalls/readNotification';
+
+interface notification {
+  type: string;
+  title: string;
+  description: string;
+  read: boolean;
+  date: Date;
+  image: string;
+  _id: string;
+}
 
 const LoggedInBar = (): JSX.Element => {
   const classes = useStyles();
+  const [unread, setUnread] = useState(false);
+  const [notificationsOpen, setNotificationsOpen] = useState(false);
+  const [notifications, setNotifications] = useState<notification[]>([
```

Comment:
```suggestion
  const [notifications, setNotifications] = useState<notification[]>([])
```
Any reason for using a mock object here?

Reply:
I just got rid of it. I was previously facing issues in TypeScript without it, but I was just able to delete it without any errors arising.